mmocr/core/evaluation/utils.py

```diff
@@ -162,35 +170,39 @@ def poly_intersection(poly_det, poly_gt, invalid_ret=0, return_poly=False):
             return poly_obj.area
     else:
         if return_poly:
-            return invalid_ret, None
+            return invalid, None
         else:
-            return invalid_ret
+            return invalid
 
 
-def poly_union(poly_det, poly_gt, invalid_ret=0, return_poly=False):
+def poly_union(poly_det, poly_gt, invalid='allow', return_poly=False):
```

Comment:
invalid has multi meanings. Can we use two parameters? We also need check the invalid inputting legality.